android/shared/src/commonMain/kotlin/io/rebble/cobble/shared/handlers/SystemHandler.kt

```diff
@@ -122,13 +122,20 @@ class SystemHandler(
         val timezone = TimeZone.currentSystemDefault()
         val now = Clock.System.now()
         val timezoneOffsetMinutes = timezone.offsetAt(now).totalSeconds.seconds.inWholeMinutes
-        Logging.i("Sending current time to watch: $now, timezone: ${timezone.id} (truncated: ${timezone.id.take(MAX_TIMEZONE_NAME_LENGTH)}), offset: $timezoneOffsetMinutes")
+   
+        val normalizedZone = timezone.id
+        if (normalizedZone.length > MAX_TIMEZONE_NAME_LENGTH) {
+            normalizedZone = timezone.id.take(MAX_TIMEZONE_NAME_LENGTH)
+            Logging.i("Time Zone ${timezone.id} exceeds maximum value length and has been truncated to ${normalizedZone}")
```

Comment:
Since this truncation 

1. is a recent addition - https://github.com/pebble-dev/mobile-app/commit/a4360399b386ae9b9c592afad617be4a3780b4c9
2. most likely is an edge case

I felt it was worth being noisy about it in this separate log, instead of trying to shoehorn it into the existing log.